recipes/stages/detection/finetune.py

```diff
@@ -4,7 +4,7 @@
     '../_base_/models/detectors/detector.py'
 ]
 
-model = dict(super_type='UnbiasedTeacher')  # Used as general framework
+model = dict(super_type='UnbiasedTeacher', pseudo_conf_thresh=0.25)  # Used as general framework
```

Comment:
I'm a little worried about if this model config is proper to be inside finetune.py. If it's just overridden in semisl.py, I think it's proper to move there, but I would still wonder if it's right place to be, because recipe file is not for model cfg file. I wanna suggest to move this config to model.py inside semi/model.py in otx. If looks hard, I can agree to leave it in recipe file.

Reply:
I understand what you're worrying about. Let me think about it

Reply:
I moved model config to otx side. Could you check it?